.github/workflows/pages.yml

```diff
@@ -33,14 +33,15 @@ jobs:
       
       - name: Build Sphinx
         run: |
-          pip install sphinx pandoc pydata_sphinx_theme nbsphinx sphinx-gallery
+          pip install "sphinx<6" pandoc pydata_sphinx_theme nbsphinx sphinx-gallery
           cd docs
           make html
+          make html

```

Comment:
This was the 'ugly hotfix' right? Do we want to keep that?

Reply:
yep...